src/Entity/User.php

```diff
@@ -27,6 +27,7 @@ class User implements UserInterface
      */
     const ROLE_USER = 'ROLE_USER';
     const ROLE_ADMIN = 'ROLE_ADMIN';
+    const ROLE_UNVERIFIED = 'ROLE_UNVERIFIED';
```

Comment:
For now we can keep the ROLE_UNVERIFIED but in the long run I think we will need to separate the notion of role and the notion of status: when you create an account you have a ROLE_USER and a STATUS_UNVERIFIED, then your status evolves but you're still a ROLE_USER. 
As I said not important right now but let's keep that in mind !

Reply:
Note